bin/main

```diff
@@ -1,60 +1,46 @@
 #!/usr/bin/env ruby
 
-puts 'WELCOME TO THIS TIC-TAC-TOE GAME: created by: '
+require_relative '../lib/board'
+require_relative '../lib/game'
+require_relative '../lib/player'
+
+def game_loop(game)
+  while game.still_playing
```

Comment:
Why `still_playing` variable? You set it as **true** and you don't modify it anywhere. What is the reason of that variable?
Here is what I suggest you
```rb
while True
...
end
```

Reply:
We favor variables over literals. It didn't change but it will come in handy when we improve on the code.